actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -252,7 +252,7 @@ def self_closing_tag_string(name, options, escape = true, tag_suffix = " />")
         def content_tag_string(name, content, options, escape = true)
           tag_options = tag_options(options, escape) if options
 
-          if escape
+          if escape && content && !content.empty?
```

Comment:
Does the lonely operator + `#present?` win us any gains?

```suggestion
          if escape && content&.present?
```